code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/code_generation/KubernetesCodeGenerator.xtend

```diff
@@ -0,0 +1,107 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base.code_generation
```

Comment:
Please have a look at `DockerFileCodeGenerator` above, because most of my comments from there also apply here.